tokenserver-db-postgres/src/models.rs

```diff
@@ -371,7 +362,7 @@ impl TokenserverPgDb {
         let mut metrics = self.metrics.clone();
         metrics.start_timer("storage.post_user", None);
 
-        diesel::sql_query(QUERY)
+        let result = diesel::sql_query(QUERY)
```

Comment:
FYI I just noticed the last commit breaking this (you can fix this later, I'm sure the tests will show it):

We switched from `SELECT LAST_INSERT_ID() AS id` to `RETURNING uid` but `results::PostUser` still assumes the old column name of `id`.

A quick fix would be `RETURNING uid as id` but you might want to consider renaming the struct field to `uid` which would be more accurate